packages/suite/src/components/suite/labeling/MetadataLabeling/definitions.ts

```diff
@@ -13,6 +13,7 @@ export interface Props {
     // override default behavior of metadata labeling element visible only on hover
     visible?: boolean;
     placeholder?: string;
+    updateFlag?: any;
```

Comment:
Could we be more specific about type here?

Reply:
I agree.

I was used to naming it `xxxCounter` and having it as a (only increasing) number. That seems to me as obvious way how to communicate the intent. 

But I am open to any naming/type that is clear enough about the intent of this prop.

Reply:
I'm not sure how to handle this, though. At this moment, the "flag" is a boolean, because I need the input to re-render when the header isScrolling (or not), but obviously, I don't care about the value itself and the input doesn't need to know what true or false means.

Reply:
@adamhavel true, I am ok to keep it.

maybe one more suggestion: what about passing `forceUpdateDependecies: any[]` an the simply spreading them into dependecy array?

Not sure if this works? But maybe it can be a good (readable) pattern how to do those forced re-renders)